server/service/osquery_utils/queries.go

```diff
@@ -150,54 +150,13 @@ limit 1
 	"os_version_windows": {
 		// Windows-specific registry query is required to populate `host.OSVersion` for Windows.
 		Query: `
-WITH dv AS (
 	SELECT
-		data
-	FROM
-		registry
-	WHERE
-		path = 'HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\DisplayVersion'
-),
-rid AS (
-	SELECT
-		data
+		os.name,
+		os.codename as display_version
+	
 	FROM
-		registry
-	WHERE
-		path = 'HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\ReleaseId'
-)
-SELECT
-	(
-		SELECT
-			name
-		FROM
-			os_version) AS name,
-	CASE WHEN EXISTS (
-		SELECT
-			1
-		FROM
-			dv) THEN
-		(
-			SELECT
-				data
-			FROM
-				dv)
-		ELSE
-			""
-	END AS display_version,
-	CASE WHEN EXISTS (
-		SELECT
-			1
-		FROM
-			rid) THEN
-		(
-			SELECT
-				data
-			FROM
-				rid)
-		ELSE
-			""
-	END AS release_id`,
+		os_version os,
+		kernel_info k`,
```

Comment:
Forgot how this works, is `k` not used?

Reply:
Oh, yeah, my bad, I grabbed the other query and removed the columns we don't need... so that table was residual.